.github/workflows/aggregator.yml

```diff
@@ -6,13 +6,19 @@ on:
       - 'main'
     paths:
       - 'aggregator/**'
+      # Check for breaking changes from contracts
+      - 'contracts/**'
       - '.github/workflows/aggregator.yml'
   pull_request:
-    branches:
-      - 'main'
     paths:
       - 'aggregator/**'
+      # Check for breaking changes from contracts
+      - 'contracts/**'
       - '.github/workflows/aggregator.yml'
+    branches-ignore:
+      # Changes targeting this branch should be tested+fixed when being merged
+      # into main
+      - contract-updates
```

Comment:
Was this disabled as the agg tests were failing/taking too long? We should run all relevant tests on any PR so the author knows if they are breaking/not breaking something.

Reply:
Discussed with Andrew, this makes it so contracts changes that break agg are ok going into `contract-updates` but not into main.